build.sbt

```diff
@@ -18,18 +17,38 @@ val commonSettings = Seq(
 
 val root = project.in(file("."))
   .doNotPublish
-  .aggregate(core)
+  .aggregate(macros, core)
+
+lazy val macros = project
+  .settings(commonSettings)
+  .settings(
+    name := "macros"
+  )
+  .dependsOnRemote(
+    "org.scala-lang" % "scala-reflect" % "2.11.8",
+    compilerPlugin("org.scalamacros" % "paradise" % "2.1.0" cross CrossVersion.full)
+  )
 
 lazy val core = project
   .configs(IntegrationTest)
   .enablePlugins(JavaServerAppPackaging)
   .settings(commonSettings)
   .settings(Defaults.itSettings)
   .settings(
+    name := "jibe",
```

Comment:
On the branch where I've been working on a launcher, I adopted the project naming convention of jibe-<module> because it makes the jar filenames prettier.